cmd/oras/discover.go

```diff
@@ -150,14 +151,24 @@ func fetchAllReferrers(ctx context.Context, repo *remote.Repository, desc ocispe
 	for _, r := range results {
 		// Find all indirect referrers
 		referrerNode := node.AddPath(r.ArtifactType, r.Digest)
+		if opts.Verbose {
+			for k := range r.Annotations {
```

Comment:
You can obtain the value directly from the map instead of `r.Annotations[k]`.
```suggestion
			for k, v := range r.Annotations {
```